way_to_home/tests/unittests/utils/testvalidators.py

```diff
@@ -0,0 +1,265 @@
+                        'start_date': '2019-02-12'}
+        result = end_date_validator(**invalid_date)
+        self.assertFalse(result)
+
+        test_date = {'end_date': '2019-02-12',
+                     'start_date': '2019-01-12'}
+        result = end_date_validator(**test_date)
+        self.assertTrue(result)
+
+    def test_time_validator(self):
+        """Method that tests the time_validator function"""
+        invalid_time = 123
+        result = time_validator(time=invalid_time)
+        self.assertFalse(result)
+
+        invalid_time = '12323fd'
+        result = time_validator(time=invalid_time)
+        self.assertFalse(result)
+
+        test_time = '21:43:23'
+        result = time_validator(time=test_time)
+        self.assertTrue(result)
+
+    def test_week_day_validator(self):
+        """Method that tests the week_day_validator function"""
+        invalid_week_day = 'fgs'
+        result = week_day_validator(week_day=invalid_week_day)
+        self.assertFalse(result)
+
+        invalid_week_day = 15
+        result = week_day_validator(week_day=invalid_week_day)
+        self.assertFalse(result)
+
+        invalid_week_day = -15
+        result = week_day_validator(week_day=invalid_week_day)
+        self.assertFalse(result)
+
+        test_week_day = 5
+        result = week_day_validator(week_day=test_week_day)
+        self.assertTrue(result)
+
+    def test_email_validator(self):
+        """Method that tests the email_validator function"""
+        invalid_email = 123
+        result = email_validator(email=invalid_email)
+        self.assertFalse(result)
+
+        invalid_email = 'bad_email'
+        result = email_validator(email=invalid_email)
+        self.assertFalse(result)
+
+        test_email = 'email@mail.com'
+        result = email_validator(email=test_email)
+        self.assertTrue(result)
+
+    def test_password_validator(self):
+        """Method that tests the email_validator function"""
+        invalid_password = 123
+        result = password_validator(password=invalid_password)
+        self.assertFalse(result)
+
+        invalid_password = 'bad_password'
+        result = password_validator(password=invalid_password)
+        self.assertFalse(result)
+
+        test_password = '123456Aa'
+        result = password_validator(password=test_password)
+        self.assertTrue(result)
+
+    def test_credentials_validator(self):
+        """Method that tests the credentials_validator function"""
+        invalid_data = {'email': 123}
+        result = credentials_validator(data=invalid_data)
+        self.assertFalse(result)
+
+        invalid_data = {'email': 123,
+                        'password': None}
+        result = credentials_validator(data=invalid_data)
+        self.assertFalse(result)
+
+        invalid_data = {'email': 123,
+                        'password': 123}
+        result = credentials_validator(data=invalid_data, update=True)
+        self.assertFalse(result)
```

Comment:
With `update=True` you just need to display tests when one of the fields will be `None` and one of the fields will not exist in the input dictionary.